cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -765,6 +769,25 @@ TYPED_TEST(MultiStepReductionTest, Mean)
             expected_value_nulls);
 }
 
+template <typename T>
+double calc_var(std::vector<T> const& v, int ddof, std::vector<bool> const& mask = {})
+{
+  auto const values = [&]() {
+    if (mask.empty()) { return v; }
+    std::vector<T> masked{};
+    thrust::copy_if(
```

Comment:
```suggestion
    std::copy_if(
```
Prefer using the STL for host containers when possible.

Reply:
Looks like `std::copy_if` does not have transform pattern like `thrust::copy_if`. 
Sorry about that. Looks like it will be easier to just change this back.

Reply:
Reverted.